package.json

```diff
@@ -27,7 +27,7 @@
     "preinstall": "npx @open-sauced/check-engines",
     "prestart": "npx @open-sauced/check-engines",
     "push": "npx @open-sauced/conventional-commit",
-    "dev": "next dev",
+    "dev": "netlify dev",
```

Comment:
Do we need to run this through netlify every time? Or can we add it as `netlify:dev`

Reply:
This can stay as `next dev` so folks can run just the Next.js dev server and if you run want to run `netlify dev` the `dev` command should already be configured.

https://github.com/open-sauced/insights/blob/netlify-edge-function/netlify.toml#L10-L13

Reply:
Updated this and added a note in the readme